payroll-processor-client/src/app/data/data-service.ts

```diff
@@ -12,16 +13,17 @@ export class DataService {
   private employees: Employee[] = [];
   private payroll: Payroll[] = [];
 
-  constructor(private http: HttpClient) {
+  private readonly apiUrl: string;
+
+  constructor(private http: HttpClient, envService: EnvService) {
```

Comment:
Sometimes we won't need the entire service as a dependency, and instead we can grab just the values we need in the constructor.

That's the case here, so I'm just getting the `apiRootUrl` for our api calls.